src/algorithms.jl

```diff
@@ -17,5 +17,5 @@ If the algorithm is `constrained` only steps of size at most the minimal delay w
 taken. If it is unconstrained, fixed-point iteration `fpsolve` is applied for step sizes
 that exceed the minimal delay.
 """
-MethodOfSteps(alg; constrained = false, fpsolve = FPFunctional()) =
+MethodOfSteps(alg; constrained = false, fpsolve = NLFunctional()) =
```

Comment:
I would argue that after this settles we probably want to default to AA. But that doesn't need to happen now.